package.json

```
@@ -71,6 +71,7 @@
     "vfile": "^5.3.7"
   },
   "devDependencies": {
+    "@lhci/cli": "^0.12.0",
```

per https://github.com/nodejs/nodejs.org/blob/main/DEPENDENCY_PINNING.md#when-pinning-dependencies this should be tilde-versions

```suggestion
    "@lhci/cli": "~0.12.0",
```

---

Removed this dependency I had added it to test forgot to remove while pushing It is done now